README.md

```diff
@@ -1,3 +1,34 @@
-# Extension Activity Monitor
+# Extension Activity Monitor [![Build Status](https://travis-ci.org/mozilla/extension-activity-monitor.svg?branch=master)](https://travis-ci.org/mozilla/extension-activity-monitor) [![Coverage Status](https://coveralls.io/repos/github/mozilla/extension-activity-monitor/badge.svg?branch=master)](https://coveralls.io/github/mozilla/extension-activity-monitor?branch=master)
 
-TBD
+This is a privileged extension for Firefox that monitors the activities of other installed extensions. The extension uses a privileged API called `activityLog` API to retrieve logs from other installed extensions.
+
+## Installation
+
+### Prerequisite
+
+- Clone / download the respository in your local machine.
+- Install [Firefox Nightly](https://www.mozilla.org/en-US/firefox/all/#product-desktop-nightly) to run this privileged extension.
+
+### Using [web-ext](https://github.com/mozilla/web-ext)
+
+- Run `npm run start:firefox` to load the extension temporarily in the Firefox Nightly.
+
+_NOTE_: You need to open Firefox Nightly previously to run the above command. If Firefox Nightly is undetectable or if you have installed it in a different location you may try the following command:
+
+```
+ web-ext run -s src -f /path/to/firefox-nightly/firefox --pref="extensions.experiments.enabled=true"
+```
+
+### Manual Installation
+
+You can follow any of the following two ways:
+
```

Comment:
For the Manual Installation it may be worth to mention explicitly that "extensions.experiments.enabled" should be set to true from about:config.

Reply:
I have updated this part to a single way for Manual Installation and mention the `extensions.experiments.enabled` part. Is this okay ?